docs/07-test-timeouts.md

```diff
@@ -22,3 +22,12 @@ test('foo', t => {
 	// Write your assertions here
 });
 ```
+
+When timeouts are set for individual tests, you can also provide an optional message.
+
+```js
+test('foo', t => {
+	t.timeout(100, 'test exceeds the expected time'); // 100 milliseconds
```

Comment:
I would show an actually useful message here. `test exceeds the expected time` adds nothing over what AVA shows by default and hence doesn't show the user why this parameter might be useful.

Reply:
Updated example, bit a still minimal but more realistic usage of the custom timeout feature.